src/instructlab/model/evaluate.py

```diff
@@ -863,4 +864,5 @@ def evaluate(
             )
     except EvalError as ee:
         print(ee.message)
+        traceback.print_exc()
```

Comment:
I would vote to move this to debug `logger.debug(ee)` since the errors returned as EvalErrors are known conditions.

Reply:
+1